spec/std/big/big_float_spec.cr

```diff
@@ -1,10 +1,19 @@
 require "spec"
 require "big"
 
+private def it_converts_to_s(value : BigFloat, str, *, file = __FILE__, line = __LINE__)
+  it "converts to #{str}", file: file, line: line do
+    value.to_s.should eq(str), file: file, line: line
+    new_value = BigFloat.new(str: str, base: 10, precision: value.precision)
+    new_value.should eq(value), file: file, line: line
+    value
```

Comment:
What's the purpose of this line?

Reply:
Will change, it used to be an assertion instead of a test case so the formal return value is no necessary here.